src/libraries/System.Runtime.Numerics/tests/BigInteger/BigNumberTools.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Numerics;
+using System.Reflection;
+
+namespace BigNumberTools
+{
+    public class Utils
+    {
+        private static TypeInfo InternalNumber
+        {
+            get
+            {
+                if (s_lazyInternalNumber == null)
+                {
+                    Type t = typeof(BigInteger).Assembly.GetType("System.Numerics.BigNumber");
```

Comment:
Usage of nameof? Something like:
```cs
Type t = typeof(BigInteger).Assembly.GetType($"{nameof(System.Numerics)}.{nameof(BigNumber)}");
```

Reply:
`BigNumberTools` is mostly copied from `BigIntTools`; If we are going to use nameof, we might as well change BigIntTools. It would be more desirable if we could combine two classes, but I haven't come up with a way to do that.
https://github.com/dotnet/runtime/blob/7f9fcd815e5efdd7a58703f2b53b611bd79a3b2e/src/libraries/System.Runtime.Numerics/tests/BigInteger/BigIntTools.cs